Model/Quote/SaveOrderBeforeSalesModelQuoteObserver.php

```diff
@@ -95,38 +100,44 @@ public function execute(\Magento\Framework\Event\Observer $observer)
     }
 
     /**
-     * @param $quote
+     * @param int $quote
      *
      * @return bool
      */
-    private function isMyParcelMethod($quote) {
+    private function isMyParcelMethod(int $quote): ?bool
```

Comment:
$quote is geen int en je returnt altijd een bool

```suggestion
    private function isMyParcelMethod(Quote $quote): bool
```